EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
+QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas, and use them as a proof of the victory. Look for Bradislovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up. 
 QUEST_JOBSTEP_20150730_003088	It seems that the envoy is dead. Return to Doppensoeldner Master and report.
-QUEST_JOBSTEP_20150803_003089	I am researching about the eternal life recently.{nl}I am investigating various documents, but it will be the best to listen from the one who is living an eternal life.{nl}
-QUEST_JOBSTEP_20150803_003090	Amjinases, they are called like that. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
-QUEST_JOBSTEP_20150803_003091	Among the Amjinas, the whereabouts of the Sculptor Tesla at the West Siauliai Woods...{nl}the Necromancer Master at Mullers Passage and the Chronomancer Master at Stele Road. I am counting on you.
-QUEST_JOBSTEP_20150803_003092	Amjinases are indeed different each other.{nl}I should think more widely and find them out.{nl}
-QUEST_JOBSTEP_20150803_003093	Anyway, thank for helping my research.{nl}As a return, I will teach you the last level of Sorcerer.
-QUEST_JOBSTEP_20150803_003094	You've come to learn high level magic of the Chronomancers.{nl}But, right now I am helping Vaidotas from the Winterspoon Family so I can't help you at the moment.{nl}
-QUEST_JOBSTEP_20150803_003095	{nl}If you are okay, could you help me investigating the Crystal Mine?{nl}If that's so, I will finish it quickly and help you.
-QUEST_JOBSTEP_20150803_003096	With this orb, explore all regions in District 1 and 2 in the Crystal Mine.{nl}If you had explored them completely in the past, then you just need to record it in the orb.
+QUEST_JOBSTEP_20150803_003089	I have recently been researching eternal life.{nl}I am investigating various documents, but it will be the best to listen from those who are already living an eternal life.{nl}
+QUEST_JOBSTEP_20150803_003090	Those that have eternal life are called Amjinases. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
+QUEST_JOBSTEP_20150803_003091	Among the Amjinases, there is the Sculptor Tesla at West Siauliai Woods...{nl}The Necromancer Master at Mullers Passage, and the Chronomancer Master at Stele Road. I am counting on you.
+QUEST_JOBSTEP_20150803_003092	These Amjinases all seem to have different reasons behind their situations.{nl}I should look harder at each...{nl}
+QUEST_JOBSTEP_20150803_003093	Anyway, thank you for helping my research.{nl}In return, I'll teach you any remaining knowledge of what it is to be a Sorcerer.
+QUEST_JOBSTEP_20150803_003094	You've come to learn high level Chronomancer magic.{nl}But, right now I am helping Vaidotas from the Winterspoon Family, so I can't help you at the moment.{nl}
```

Comment:
`high level Chronomancer magic.{nl}But, right`
to
`high level Chronomancer magic, {nl}but right`
for the sake of flow and connecting the two thoughts together